### PR TITLE
doc: Update README, add CONTRIBUTION & SECURITY

### LINKED ISSUE #2041: README.md Badges Are Broken or Incorrect
The badges at the top of the README.md file misrepresent the actual state of the project.  They should either be updated or removed.  We shouldn't mislead anyone who visits the repo.

### LINKED ISSUE #2042: Update README.md To Include NEAR Mission
The current README includes a few sentences about what NEAR Protocol does, or hopes to do; however, I suspect it might be better to add an actual mission statement that is consistent with all the other representations of the mission in other documents.

### LINKED ISSUE #2043: Project Status Section of README.md is Misleading
Our github repositories should be the most up-to-date source of truth.  Currently the project status contains misleading information about our test and dev networks.  For example:

> High level milestones:
> 
>  DevNet: a tool with fully working State Transition + WebAssembly.
>  AlphaNet: Multi-node smart-contract platform.
>  BetaNet: Added economics and enhanced security.
>  (In progress) Nightshade: Added sharding and staking.
>  TestNet: Added governance module, ready to launch as MainNet
>  MainNet: Launched full blockchain.

Doesn't reflect reality.

### LINKED ISSUE #2044: Running a Local Node Documentation Fails
Following the instructions linked to from the README.md to run a local node does not work: 
 https://docs.nearprotocol.com/docs/local-setup/running-testnet 

By following those instructions, the developer's node will fail whenever staging and master are out of sync, which is almost always the case as our release cycles and practices leave branches/networks out of version sync.

### LINKED ISSUE #2046: Add Contribution Guidelines to Github:  CONTRIBUTING.md
Need to create/add our contributor guidelines to CONTRIBUTING.md in nearcore.

### PR DESCRIPTION
**Pull request by ilblackdragon**
Long overdue refresh of README.
Also added CONTRIBUTION (from CW discussion) and SECURITY disclosure policy and process.

Fixes #2041
Fixes #2042
Fixes #2043
Fixes #2044
Fixes #2046

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,46 +9,77 @@
...
-[![NEAR Studio](https://github.com/nearprotocol/NEARStudio/blob/master/demos/guest_book.gif)](https://studio.nearprotocol.com)
+To achieve this purpose, *NEAR* provides a developer platform where developers and entrepreneurs can create apps that put users back in control of their data and assets, which is the foundation of ["Open Web" movement][open-web-url].
 
+One of the components of *NEAR* is NEAR Protocol, an infra for server-less applications and smart contracts powered by blockchain.
+NEAR Protocol is built to deliver usability and scalability of modern PaaS like Firebase at fraction of prices that blockchains like Ethereum charge.
 
-## Status
+*NEAR* overall provides wide range of tools for developers to easily build applications:
+ - [JS SDK][js-api] to connect ot NEAR Protocol from your applications.
+ - [Rust][rust-sdk] and [AssemblyScript][as-sdk] SDKs to write sever-less functions & smart contracts.
```
Comment (frol) [inline README.md]: typo: "sever-less"

File: README.md [hunk 2]
```diff
@@ -9,46 +9,77 @@
...
 
-[![NEAR Studio](https://github.com/nearprotocol/NEARStudio/blob/master/demos/guest_book.gif)](https://studio.nearprotocol.com)
+To achieve this purpose, *NEAR* provides a developer platform where developers and entrepreneurs can create apps that put users back in control of their data and assets, which is the foundation of ["Open Web" movement][open-web-url].
 
+One of the components of *NEAR* is NEAR Protocol, an infra for server-less applications and smart contracts powered by blockchain.
+NEAR Protocol is built to deliver usability and scalability of modern PaaS like Firebase at fraction of prices that blockchains like Ethereum charge.
 
-## Status
+*NEAR* overall provides wide range of tools for developers to easily build applications:
+ - [JS SDK][js-api] to connect ot NEAR Protocol from your applications.
```
Comment (frol) [inline README.md]: I would frame it as:

> Public JSON-RPC API with a first-party client library available for JavaScript and TypeScript.

Comment (ilblackdragon) [reply README.md]: Hm, there is a bunch more stuff than just JSON RPC. Like Singers, KeyStores and Wallet support.

Comment (frol) [reply README.md]: Fair enough. Your last update sound more accurate :+1:

**Pull request merged**